src/social-web/routes/feed/inspector.tsx

```diff
@@ -112,6 +118,24 @@ export default function FeedInspector( { id, onClose }: FeedInspectorProps ) {
 					{ ( post.content?.rendered || post.excerpt?.rendered ) && (
 						<RenderHTML html={ post.content?.rendered || post.excerpt?.rendered || '' } />
 					) }
+					{ tags && tags.length > 0 && onTagClick && (
+						<div className="activitypub-inspector-tags">
+							{ tags.map( ( tag: any ) => (
```

Comment:
```suggestion
							{ tags.map( ( tag: Term ) => (
```
Maybe it would be good to review the file for stronger typing